### DISCUSSION TITLE
Does chunking affect loading for isel / sel

### DISCUSSION THREAD

**Discussion by ahuang11** [pydata/xarray#4605] [Q&A] [answered]
If my files are split across datetimes:
```
2017050112.nc
2017050212.nc
...
2020050212.nc
```

Is it better to chunk before loading a selection:
```
import xarray as xr
ds = xr.tutorial.open_dataset('air_temperature')
ds.chunk({'lat': 1, 'lon': 1}).isel(lat=0, lon=0).load()
```

Or directly
```
import xarray as xr
ds = xr.tutorial.open_dataset('air_temperature')
ds.isel(lat=0, lon=0).load()
```

**Comment by andersy005** [comment, marked as answer]
Going through Dask's schedulers always adds some overhead. 


![Screen Shot 2021-11-11 at 8 34 34 PM](https://user-images.githubusercontent.com/13301940/141404615-e1a1c8c5-3ad7-41c2-8521-20cc48b29be0.png)

So, If you don't need to perform out-of-core computation on large arrays, you might be better off without dask/chunking
